src/ducks/Watchlists/Actions/Edit/EditForm/index.js

```diff
@@ -138,7 +159,18 @@ const EditForm = ({
             name='description'
             className={styles.textarea}
             onChange={onTextareaChange}
-            defaultValue={formState.description || ''}
+            defaultValue={formState.description}
+            placeholder='Add a description'
+          />
+        )}
+        {isOpen && (
+          <Assets
+            watchlist={watchlist}
+            onChange={items => {
+              const _listItems = items.map(l => ({ projectId: parseInt(l.id) }))
+              setFormState(state => ({ ...state, listItems: _listItems }))
+              checkIsTouched('listItems', _listItems)
```

Comment:
`[D-0-1]` `_` prefix is redundant since `listItems` is not used in this scope, it's okay to shadow it by naming this variable the same